AnnoyancesFilter/sections/tweaks.txt

```diff
@@ -1457,7 +1458,7 @@ codedosa.com#$#html { user-select: auto!important;-webkit-touch-callout: default
 open.spotify.com#$#span[dir="auto"][draggable="true"] { user-select: auto!important;-webkit-touch-callout: default!important;-webkit-user-select: auto!important;-moz-user-select: auto!important;-ms-user-select: auto!important; }
 kakaowebtoon.com,webtoon.kakao.com#$#img[class^="ViewerImage_image"] { pointer-events: auto!important; }
 lezhinus.com,lezhin.com#$#body * { pointer-events: auto!important; }
-appofmirror.com#$#body * { user-select: auto!important;-webkit-touch-callout: default!important;-webkit-user-select: auto!important;-moz-user-select: auto!important;-ms-user-select: auto!important; }
+toptoon.com,appofmirror.com#$#body * { user-select: auto!important;-webkit-touch-callout: default!important;-webkit-user-select: auto!important;-moz-user-select: auto!important;-ms-user-select: auto!important; }
```

Comment:
It doesn't work on my end. Check this
```
toptoon.com,appofmirror.com#$#html * { user-select: auto !important; -webkit-touch-callout: default !important; -webkit-user-select: auto !important; -moz-user-select: auto !important; -ms-user-select: auto !important; }
```

Reply:
On Firefox Dev 95.0b2, my rule works.
But, It seems that your rule is better.
https://github.com/AdguardTeam/AdguardFilters/pull/98854/commits/05c8922fe869c4831b5026f5d456983e3618fa3e